### PR TITLE
Implement sending a notification to a specific user's mail

### PR DESCRIPTION
**Pull request by bogdanmilojevic**
- Add UserDto to OrderRequest and OrderCreatedEvent
- Update sendHtmlMailMessage method in notification service to send order confirmation to mail from OrderCreatedEvent

### CODE REVIEW

File: order-service/src/main/java/com/microcommerce/orderservice/data/entity/Order.java [hunk 1]
```diff
@@ -33,6 +34,9 @@ public class Order {
     @Column(name = "order_number")
     private String orderNumber;
 
+    @Column(name = "user_id")
```
Comment (ilicstef94) [inline order-service/src/main/java/com/microcommerce/orderservice/data/entity/Order.java:37]: First to merge Bojana's PR for drawing the base model and then to rebase changes from develop in order to expand the base model with the user id field.

File: order-service/src/main/java/com/microcommerce/orderservice/service/OrderService.java [hunk 2]
```diff
@@ -81,13 +74,20 @@ public ApiResponse<OrderResponse> createOrder(OrderRequest orderRequest) {
...
 
-    private void produceOrderCreatedEvent(OrderCreatedEvent orderCreatedEvent) {
+    private void produceOrderCreatedEvent(List<EventOrderItemDto> orderItems, String orderNumber, UserDto user) {
+        var orderCreatedEvent = OrderCreatedEvent.builder()
+                .orderItems(orderItems)
+                .orderNumber(orderNumber)
+                .user(user)
+                .build();
+
         var completableFuture = kafkaTemplate.send("notification-topic", UUID.randomUUID(), orderCreatedEvent);
```
Comment (ilicstef94) [inline order-service/src/main/java/com/microcommerce/orderservice/service/OrderService.java:89]: Define this configuration value in application yaml file.

**Review by ilicstef94 — CHANGES_REQUESTED** [review]

File: order-service/src/main/java/com/microcommerce/orderservice/service/OrderService.java (hunk 2 above)
Comment (bogdanmilojevic) [reply order-service/src/main/java/com/microcommerce/orderservice/service/OrderService.java]: Fixed

File: order-service/src/main/java/com/microcommerce/orderservice/data/entity/Order.java (hunk 1 above)
Comment (bogdanmilojevic) [reply order-service/src/main/java/com/microcommerce/orderservice/data/entity/Order.java:37]: Done

**Review by ilicstef94 — APPROVED** [review]

**Pull request merged**